src/components/shared/Error/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { AlertTitle, Box } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import AlertBox from '../AlertBox';
+import Message from './Message';
+import { ErrorDetails } from './types';
+
+export interface ErrorProps {
+    error?: ErrorDetails;
+    hideTitle?: boolean;
+    noAlertBox?: boolean;
+}
+
+function Error({ error, hideTitle, noAlertBox }: ErrorProps) {
+    if (!error) {
+        return null;
+    }
+
+    if (noAlertBox) {
+        return (
+            <Box sx={{ p: 1, width: '100%' }}>
+                <Message error={error} />
+            </Box>
+        );
+    }
```

Comment:
New functionality... allow this to be used with the entire AlertBox wrapping it. Mainly just for the header errors in create/edit